docs/reference/module-api/module-overview.md

```diff
@@ -817,27 +759,6 @@ Extend the behavior of existing event handlers (set in `handlers`) in the `exten
 
 Each extended event handler should accept the original function as `_super` followed by its original arguments. Extended handlers will be matched with base class handlers using the same server-side event *and* the same handler name.
 
-```javascript
-// modules/featured-product/index.js
-module.exports = {
-  extend: 'product',
-  // ...
-  handlers(self) {
-    return {
-      // Responds to `beforeInsert` when emitted by the `product` module
-      beforeInsert: {
-        async applyTax(_super, req, piece) {
-          // Add a $2.50 charge before applying taxes.
-          piece.price = piece.price + 2.5;
-
-          await _super(req, piece);
-        }
-      }
-    }
-  }
-};
-```
-
 ### `queries(self, query)`
 
 The `queries` function is part of the Apostrophe API for querying the database. It takes two arguments: the module (`self`) and the query that will be called (`query`). It must return an object. That object can have two properties:
```

Comment:
This sounds very strongly like something you call (it is "part of the Apostrophe API for querying the database"). That is going to be confusing.

I think it should say:

The `queries` section is used to extend the features of Apotrophe's API for querying the database.

Reply:
addressed in #47